lib/services/systemd.h

```diff
@@ -16,8 +16,13 @@
  * Copyright (C) 2012 Andrew Beekhof <andrew@beekhof.net>
  */
 
+#ifndef SYSTEMD__H
+#  define SYSTEMD__H
+
 G_GNUC_INTERNAL GList *systemd_unit_listall(void);
 G_GNUC_INTERNAL int systemd_unit_exec(svc_action_t * op);
 G_GNUC_INTERNAL gboolean systemd_unit_exists(const gchar * name);
 G_GNUC_INTERNAL gboolean systemd_unit_running(const gchar * name);
 G_GNUC_INTERNAL void systemd_cleanup(void);
+
+#endif  /* SYSTEMD__H */
```

Comment:
Certainly it's good that there are no conflicts now, but I meant that SYSTEMD__H seems like the kind of name that someone else might easily use in the future, causing us sudden strange build errors. I just think it's better to make it less generic -- SERVICES_SYSTEMD__H is consistent with other headers we use, and less likely to be used by someone else. It would be even better if we consistently used a unique prefix like CRMD_ or PCMK_, but I think SERVICES_SYSTEMD__H is unlikely enough.